app/models/fund.rb

```diff
@@ -2,4 +2,5 @@ class Fund < ApplicationRecord
   validates_presence_of :name
   belongs_to :organisation
   has_one :activity, as: :hierarchy
+  has_many :transactions, as: :hierarchy
```

Comment:
I think this is going to confuse folks (and possibly Rails) - we now have *two* polymorphic associations (`Activity` and `Transaction`) with the same `as:` name.

Reply:
Agree; I was adding hierarchy as requested but maybe we need another naming session